Core/TimberApi/LocalizationSystem/LocalizationPatcher.cs

```diff
@@ -25,8 +25,13 @@ public static void GetLocalizationPatch(string localizationKey, ref IDictionary<
             IDictionary<string, string> localization = LocalizationFetcher.GetLocalization(localizationKey);
             try
             {
-                __result.AddRange(localization);
+                // __result.AddRange(localization);
                 TimberApi.ConsoleWriter.Log($"Loaded {localization.Count} custom labels");
+
+                foreach (var (key, value) in localization)
```

Comment:
I'm 99% certain this only is able to replace original game text, but not modded game text.

Reply:
You are right, oups, I'll do the change in few hours !

Reply:
Just a heads up, It might not be a great idea to do it in the patch because of extra loop/dependency order. Thanks in advance! :D

Reply:
What do you mean? In all cases it will be in the patch

Reply:
I mean that the change should probably be in `LocalizationFetcher` instead of `LocalizationPatcher`. But it will indeed be both in the overall patch.

Reply:
@KYPremco updated :)